out/lock_pool.go

```diff
@@ -63,6 +66,7 @@ func (lp *LockPool) ClaimLock(lock string) (Version, error) {
 
 		return false, nil
 	})
+	fmt.Fprintf(lp.Output, "\nclaimed!\n")
```

Comment:
@vito note that I am obliged to:

1. add a `\n` at line 50, otherwise the first timestamp is always (wrongly updated). This is why my first PR was very simple (but it always worked ;-)
2. add a `\n` at the beginning of line 69, otherwise the `claimed!` will be printed in the same line as the dots